.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,31 @@
+# .github/workflows/deploy-preview.yml
+
+name: Deploy to Preview Channel
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./example
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "12.x"
+      - uses: subosito/flutter-action@v1
+        with:
+          channel: stable

```

Comment:
I think the `stable` channel is the _default_, so you don't really need to be explicit.

:rocket: Advanced would be to offer a _Workflow Input Parameter_ that defaults to `stable`, but can be overridden to be another channel, or a version, and then use that.

Reply:
Fixed!